tensorflow/python/kernel_tests/check_ops_test.py

```diff
@@ -302,6 +302,44 @@ def test_returns_none_with_eager(self):
       x = check_ops.assert_none_equal(t1, t2)
       assert x is None
 
+  def test_error_message_eager(self):
```

Comment:
Can you make the test less strict? I foresee many many changes to this test as small details of the error messasge might change in the future.

How about just checking for the presence of the tensor contents?

Reply:
Sure, I'd be happy to. Will push another commit tomorrow.

Reply:
Changes pushed. Regression tests pass on my test machine.